airflow/settings.py

```diff
@@ -104,6 +104,7 @@
     "up_for_reschedule": "turquoise",
     "up_for_retry": "gold",
     "upstream_failed": "orange",
+    "shutdown": "black",
```

Comment:
```suggestion
    "shutdown": "blue",
```

Let's return historical value for shutdown.
This state, as well as color values removed into https://github.com/apache/airflow/pull/33746/files the but lately constants brings back by https://github.com/apache/airflow/pull/34063